### DISCUSSION TITLE
How to enable borg backup after installation -- initially disabled

### DISCUSSION THREAD

**Discussion by tansowny-tech** [nextcloud/all-in-one#5471] [Questions] [answered]
I disabled the borg backup with an environment variable in Docker Compose on my initial installation but want to enable it now without losing anything. Is there a way to do this without completely resetting my install?

**Comment by szaimen** [comment, marked as answer]
Hi, Have you tried to set the value to anything other than true?

**Reply by scott-tansowny** [reply]
I removed that line from my docker-compose and ran:

```
docker stop nextcloud-aio-mastercontainer
docker rm nextcloud-aio-mastercontainer
docker compose up -d
```

What am I missing?

**Reply by szaimen** [reply]
Removing the value is not sufficient. You need to set it to something else than true

**Reply by tansowny-tech** [reply]
Thank you very much, that worked!
